### DISCUSSION TITLE
Elasticsearch integrations

### DISCUSSION THREAD

**Discussion by 8bitjoe** [Security-Onion-Solutions/securityonion#10047] [2.4] [answered]
Greetings, 

I had a question about the new release, since there will be support for elastic agent will there also be support for elastic search integrations? Currently i managed to add integrations through kibana in the elastic search manager of a distributed deployment but the logs are being saved in the /nsm of the manager node, i was hoping that with this new elastic agent support of Sec onion 2.4 elastic search integrations would be done with the search node. 
Is that the case? or maybe you could give me a recommendation on how to add integrations to the elastic search container in the search node, since there is no way to make changes on it directly with kibana. I'm not sure how i would add an integration from the API.

**Comment by TOoSmOotH** [comment, marked as answer]
Elastic is a single cluster now. What integration are you talking about?

**Reply by 8bitjoe** [reply]
ahh ok, thats great then, i use a sonicwall integration to bring the firewall logs into elastic search, however right now with my 2.3 distributed deployment all that data is being saved on my manager node. i guess with elastic being a single cluster now that resolves that issue so i will go ahead and try out the beta release.
I have just one final question, if i try out this beta will i be able to upgrade to the production release once it comes out?

**Reply by TOoSmOotH** [reply]
From the blog post:

"Upgrades from this 2.4 Beta release to anything else will not be supported. Starting in RC1 we will support soup to upgrade 2.4 grids."

The pipeline is getting a bit of an overhaul after Beta 1. Currently elastic agents are sending their data directly to Elastic but starting in Beta 2 we will be sending it to Logstash. This was due to a bug in Elastic that they are addressing in the next release of ES.
